qemu/tests/cfg/fio_windows.cfg

```diff
@@ -2,12 +2,13 @@
     type = fio_windows
     only Windows
     test_timeout = 600
-    install_path = "C:\fio"
     log_file = "C:\fio_log.txt"
     i386, i686:
-        install_cmd = "msiexec /a DRIVE:\fio-x86.msi /qn TARGETDIR=${install_path}"
+        install_path = "C:\Program Files (x86)\fio"
+        install_cmd = "msiexec /i DRIVE:\fio-2.2.10-x86.msi /qn"
     x86_64:
-        install_cmd = "msiexec /a DRIVE:\fio-x64.msi /qn TARGETDIR=${install_path}"
-    config_cmd = 'setx -m path "%PATH%;${install_path}\fio;"'
-    fio_cmd = '"${install_path}\fio\fio.exe" --name=fiotest --rw=randrw --iodepth=4 --bs=4k'
```

Comment:
there is fio in install_path, so need to drop this one, otherwise could not find the file